contracts/logic/Validation.sol

```diff
@@ -18,9 +17,9 @@ library Validation {
    * @param amount Deposit amount
    **/
   function validateDeposit(DataStruct.ReserveData storage reserve, uint256 amount) internal view {
-    if (amount == 0) revert MoneyPoolErrors.InvalidAmount(amount);
-    if (reserve.isPaused == true) revert MoneyPoolErrors.ReservePaused();
-    if (reserve.isActivated == false) revert MoneyPoolErrors.ReserveInactivated();
+    require(amount != 0, 'InvalidAmount');
+    require(reserve.isPaused != true, 'ReservePaused');
+    require(reserve.isActivated != false, 'ReserveInactivated');
```

Comment:
`isPaused` and `isActivated` are boolean.
I think, `!reserve.isPaused` and `reserve.isActivated` are enough.

Reply:
Great !